README.md

```diff
@@ -36,3 +36,45 @@ npm run build
 You can preview the production build with `npm run preview`.
 
 > To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+
+---
+
+# Actual README Content
+
+## Ubiquitous Language
+- Campaign: A story co-written by a game master and their players about the non-player characters the GM runs an the player characters the players run.  Could be based in any TTRGP series or home brew game.
+- Session: A period of continuous time when the GM and players got together to advance the story/game.
+- Game master: the user who creates a campaign and owns the right to dole out or restrict editing permissions.
```

Comment:
Thoughts on permissions...
- A TTRPG group is generally an environment of trust.  Therefore, we may want to be on the generous side with permissions, and counterbalance it via a public audit trail.
- Certain things may have default permissions.  For example, if Marie owns a given quest, then she can mark its goals as completed, add milestones, etc.  Maybe things like deletion and retroactive editing, and advancing to the next session, are reserved for the GM.

This may all be way off from what actual TTRPG players would expect, but I just want to plant a seed on this thinking.

Reply:
Ignore.  I should read the readme.

Reply:
Also:  I pictured a GM who restricts permissions in order to maintain standards of narrative flourish 😄

Reply:
Another critical use-case from the player side for managing visibility and editing:

Players often have secret goals.  Especially at the start of the game.  The rest of the party isn't privy to those goals unless the player's character admits to those goals or something in the story line forces a reveal.  For the example data, I have a case of that with Cordelia.

She is a masquerade breacher, which is a serious offense.  She has not told Lyle.  Cordelia's player, Bob, may not want Joe to know about the fact (or the quest associated with it) so that Joe can more easily play Lyle's ignorance (as both player and player character are ignorant).  However, the GM, Tom, 100% has to know everything about Cordelia that Bob wants the story to focus on.  Sharing Cordelia's "Not so Flawless" quest only with the GM lets the GM decide that the Prince knows Cordelia's secret, so now it's a plot element.  Thus, Cordelia's secret could be revealed through many story mechanisms, and it will feel like a big reveal to Joe as a player.